### PR TITLE
Fix dialog calculations & issues with re-opening emoji keyboard

### PR DESCRIPTION
**Pull request by mario**
This PR does two things:

a) Fixes issues introduced by https://github.com/vanniktech/Emoji/pull/373
b) Fixes dialog emoji view calculations so it should now work properly

Do note that the setHeight thing introduced by PR 373 is probably broken by this PR, but on the other hand it never worked properly and only introduced regressions.

I plan to look into salvaging the functionality of that PR in a followup one.

WIP atm.
Signed-off-by: Mario Danic <mario@lovelyhq.com>

### CODE REVIEW

**Comment by mario** [comment]
Not entirely happy (as it's partly a workaround) but it's the best we have for now.

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java [hunk 1]
```diff
@@ -138,10 +136,9 @@
         }
       }
     });
-
     popupWindow.setContentView(emojiView);
+
     popupWindow.setInputMethodMode(PopupWindow.INPUT_METHOD_NOT_NEEDED);
-    popupWindow.setBackgroundDrawable(new BitmapDrawable(context.getResources(), (Bitmap) null)); // To avoid borders and overdraw.
```
Comment (vanniktech) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: How come we can remove this?

**Comment by mario** [comment]
@vanniktech what commit did you use? I noticed I forgot one a few minutes ago.

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java (hunk 1 above)
Comment (mario) [reply emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: Because we do not pass the context to the PopupWindow afaik.

**Comment by vanniktech** [comment]
I've used the last one (9f27617). Using Pixel 3XL mit Android 10

**Comment by mario** [comment]
Android 10 is special :) Let me look into it.

**Comment by vanniktech** [comment]
Is there any update on this? I'd like this to be part of the next release.

**Comment by mario** [comment]
I'm working on it - it's not trivial.

**Comment by mario** [comment]
Closing in favor of a new PR.

**Comment by nAkhmedov** [comment]
@mario In which version are you planning to apply this fix?

**Comment by mario** [comment]
0.7

**Comment by vanniktech** [comment]
Use the snapshot for now

**Pull request closed without merging**